crates/wadm/src/server/handlers.rs

```diff
@@ -23,10 +21,6 @@ use crate::{model::StoredManifest, publisher::Publisher};
 
 use super::{parser::parse_manifest, storage::ModelStorage, ManifestNotifier};
 
-const JSON_SCHEMA: &str = include_str!("../../oam.schema.json");
```

Comment:
Since you're removing this, can you also remove this file and the GH Actions check that makes sure this is in sync with the top level schema?

Reply:
Other than the nondescript errors here, is there a good reason why we should not have a JSON schema embedded? I'm okay with this as long as the validation is done client-side to reasonably avoid errors, but I do like the assertion that an incoming manifest is valid

Reply:
So the utoipa crate is actually validating that the manifest is correct, has all the right fields, etc.

Having not thought about this for ~2 weeks, I'm wondering if it actually may make more sense to revert back to using a jsonschema, but instead of using a static file, actually generate the schema from our model using [schemars](https://github.com/GREsau/schemars)